internal/compiler-interface/src/main/contraband/incremental.json

```diff
@@ -124,29 +117,6 @@
           ]
         },
         {
-          "name": "antStyle",
```

Comment:
I see, my mistake. I hadn't realised the damage had already been done. My apologies.